homeassistant/components/swiss_public_transport/services.py

```diff
@@ -42,9 +41,7 @@
 )
 
 
-def async_get_entry(
-    hass: HomeAssistant, config_entry_id: str
-) -> config_entries.ConfigEntry:
+def async_get_entry(hass: HomeAssistant, config_entry_id: str) -> ConfigEntry:
```

Comment:
Wy not return `SwissPublicTransportConfigEntry` directly?
It would remove the need to cast below.

Reply:
I see you updated it directly at the same time as my comment went it.
You can still remove the cast now.

Reply:
right